.github/workflows/xcube_workflow.yaml

```diff
@@ -14,11 +14,13 @@ jobs:
   unittest:
     runs-on: ubuntu-latest
     env:
-      NUMBA_DISABLE_JIT: 1
+      NUMBA_DISABLE_JIT: 0
```

Comment:
We could also remove this part completely, depending if we want to have this switch available or not. What do you think?

Reply:
I think we should keep it -- doesn't do any harm, confirms that numba is not disabled, and saves us looking up the variable name if we need to temporarily disable it for testing in future.

Reply:
Yes, keep it.